bears/general/DuplicateFileBear.py

```diff
@@ -0,0 +1,36 @@
+import filecmp
+import itertools
+import os.path
+
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.results.Result import Result
+
+
+class DuplicateFileBear(GlobalBear):
+    LANGUAGES = {'All'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Duplicates'}
+
+    def run(self):
+        """
+        Checks for Duplicate Files using filecmp module
+        of Standard Library
+        """
+
+        unique_tuples = [fileTuple for fileTuple in
+                         itertools.combinations(self.file_dict, 2)]
+
+        if not unique_tuples:
+            yield Result(self, 'You included no files '
+                         'or only a single file.')
```

Comment:
Well, DuplicateFileBear with only a single file is IMO a config error, as it is doing nothing so the "No duplicates found" is misleading as nothing was actually attempted.